src/config/dicom/manualBlacklist.ts

```diff
@@ -0,0 +1,373 @@
+  'RTImageDescription',
+  'RTImageFrameContextSequence',
+  'RangeModulatorDescription',
+  'RangeShifterDescription',
+  'ReasonForChannelOmissionDescription',
+  'ReceiverNotes',
+  'ReconstructionDescription',
+  'ReferenceLocationDescription',
+  'ReferencedStructuredContextSequence',
+  'ReferencedTextureSequence',
+  'ReportStatusCommentTrial',
+  'RequestedProcedureDescriptionTrial',
+  'RequestedSeriesDescriptionCodeSequence',
+  'ResourceDescription',
+  'SOPAuthorizationComment',
+  'SegmentDescription',
+  'SeriesDescriptionCodeSequence',
+  'SetupDeviceDescription',
+  'SetupImageComment',
+  'SetupReferenceDescription',
+  'SourceDescription',
+  'SpecimenDescriptionSequence',
+  'SpecimenDescriptionSequenceTrial',
+  'SpecimenDescriptionTrial',
+  'StrainDescription',
+  'StructuredDisplayTextBoxSequence',
+  'SubstanceAdministrationNotes',
+  'SurfaceComments',
+  'SurfaceModelDescriptionSequence',
+  'SurfaceProcessingDescription',
+  'SystemStatusComment',
+  'TestResultComment',
+  'TextColorCIELabValue',
+  'TextObjectSequence',
+  'TextStyleSequence',
+  'TextValue',
+  'TextureLabel',
+  'TherapyDescription',
+  'ThreatCategoryDescription',
+  'TrackSetDescription',
+  'TransactionStatusComment',
+  'TransformDescription',
+  'TreatmentStatusComment',
+  'TreatmentTerminationDescription',
+  'USImageDescriptionSequence',
+  'UnformattedTextValue',
+  'VitreousStatusDescription',
+  'WedgeDescription',
+  'WindowCenterWidthExplanation',
+  'AssigningAgencyOrDepartmentCodeSequence',
+  'ClinicalFractionNumber',
+  'ClinicalTrialTimePointTypeCodeSequence',
+  'ConsentForClinicalTrialUseSequence',
+  'FilterByOperator',
+  'OphthalmicPatientClinicalInformationLeftEyeSequence',
+  'OphthalmicPatientClinicalInformationRightEyeSequence',
+  'PatientClinicalTrialParticipationSequence',
+  'PerformedStationClassCodeSequence',
+  'RTPhysicianIntentIndex',
+  'RTPhysicianIntentInputInstanceSequence',
+  'RTPhysicianIntentPredecessorSequence',
+  'RTPhysicianIntentSequence',
+  'ReferencedRTPhysicianIntentIndex',
+  'ReferencedRTPhysicianIntentSequence',
+  'RequestingPhysicianIdentificationSequence',
+  'ScheduledStationClassCodeSequence',
+  'SecondaryReviewerName',
+  'CalciumScoringMassFactorPatient',
+  'ColumnAngulationPatient',
+  'DataCollectionCenterPatient',
+  'DistanceSourceToPatient',
+  'GroupOfPatientsIdentificationSequence',
+  'ImageOrientationPatient',
+  'ImagePositionPatient',
+  'IssuerOfPatientIDQualifiersSequence',
+  'NumberOfPatientRelatedInstances',
+  'NumberOfPatientRelatedSeries',
+  'NumberOfPatientRelatedStudies',
+  'PatientAlternativeCalendar',
+  'PatientBirthDateInAlternativeCalendar',
+  'PatientBodyMassIndex',
+  'PatientBreedCodeSequence',
+  'PatientDeathDateInAlternativeCalendar',
+  'PatientEquipmentRelationshipCodeSequence',
+  'PatientEyeMovementCommandCodeSequence',
+  'PatientEyeMovementCommanded',
+  'PatientGantryRelationshipCodeSequence',
+  'PatientIdentityRemoved',
```

Comment:
This is a breaker if blacklisted (see other logic in code).
Could you do a manual review of the list to avoid these?